package com.exception.handling1;

public class ThrowDemo4 {

	public static void main(String[] args) {
		// TODO Auto-generated method stub
		
//		Exception in thread "main" java.lang.Error: Unresolved compilation problem: 
//			No exception of type ThrowDemo4 can be thrown; an exception type must be a subclass of Throwable
//
//			at javalearning/com.exception.handling1.ThrowDemo4.main(ThrowDemo4.java:8)

		
//		throw new ThrowDemo4();

	}

}
